docs/deployment_guide/partner_editable/deployment_options.adoc

```diff
@@ -1,7 +1,18 @@
 // Edit this placeholder text as necessary to describe the deployment options.
 
-This solution provides the following deployment options:
+This template provides the following deployment options:
 
 * https://fwd.aws/wNB78?[Deploy {partner-product-short-name}^]. This builds a new workflow consisting of an Amazon EventBridge rule, filtering Lambda function, SOCI index generator Lambda function, and IAM resources.
```

Comment:
this is no longer valid. There are no deployment options in the current template. Instead, we rely on users to set the regions that they wishj to deploy in their taskcat.yaml file and use the correct regional creds to deploy the template. 

This section can be updated to describe more about how to update configurations to support different regions